crc_folding.c

```diff
@@ -275,13 +275,17 @@ ZLIB_INTERNAL void crc_fold_copy(deflate_state *z_const s,
 {
     unsigned long algn_diff;
     __m128i xmm_t0, xmm_t1, xmm_t2, xmm_t3;
+    char buf[sizeof(__m128i)] __attribute__((aligned(sizeof(__m128i)))) = { 0 };
```

Comment:
Note: if desired, I can easily change this into code not depending on the `aligned` attribute.